bin/emqx

```diff
@@ -124,6 +124,20 @@ generate_config() {
         APPCONF=`relx_nodetool mergeconf $RUNNER_ETC_DIR/emqx.conf $RUNNER_ETC_DIR/plugins $RUNNER_DATA_DIR/configs`
         replace_env_in_conf
         CONFIG_ARGS=`$CUTTLEFISH_COMMAND_PREFIX -c $APPCONF generate`
+
```

Comment:
@emqplus 
The comments start from line 117 to line 121 seem to be outdated now?
Because I can not find anywhere in our code base which makes use of `vm_args` parameter.